package.json

```diff
@@ -16,7 +16,9 @@
     "onCommand:mdb.connectWithURI",
     "onCommand:mdb.addConnection",
     "onCommand:mdb.addConnectionWithURI",
+    "onCommand:mdb.disconnect",
     "onCommand:mdb.removeConnection",
+    "onCommand:mdb.launchShell",
```

Comment:
`launchShell` sounds like a shell will be opened as a separate window, won't it? Maybe `openShell` or `showShell` or `mongodbShell` would be more accurate?

Reply:
Ah - I like `openShell`. good suggestion.